pkgs/by-name/po/pokemon-colorscripts/package.nix

```
@@ -0,0 +1,46 @@
+{
+  coreutils,
+  fetchFromGitLab,
+  lib,
+  python3,
+  stdenv,
+}:
+
+stdenv.mkDerivation {
+  pname = "pokemon-colorscripts";
+  version = "unstable-2025-06-25";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning
```suggestion
  version = "0-unstable-2025-06-25";
```